domain/account/usecase.go

```diff
@@ -7,8 +7,8 @@ import (
 type UseCase interface {
 	CreateAccount(account entities.AccountInput) (entities.Account, error)
 	ListAll() ([]entities.Account, error)
-	Deposit(CPF string, amount float64) error
-	Withdraw(CPF string, amount float64) error
+	Deposit(CPF string, amount float64) (entities.TransactionOutput, error)
+	Withdraw(CPF string, amount float64) (entities.TransactionOutput,error)
```

Comment:
considering these outputs are not exactly entities, but outputs from the usecases, they should be in the same package as the `UseCase` interface